views/aggregate-flow-metadata.json

```diff
@@ -2594,14 +2646,18 @@
                     }
                 }
             },
-            "install_command": "pip install prefect-kubernetes",
```

Comment:
Are the location of the keys determinate now?

Reply:
are you asking about sorting?

Reply:
Yes